### DISCUSSION TITLE
questions about Amount and Value

### DISCUSSION THREAD

**Discussion by VladimirAlexiev** [eprocurementontology/eprocurementontology#521] [Q&A]
1. Class `Amount` is described to have props `epo:hasAmountValue, epo:hasCurrencyID`.
But further prop definitions state that these are also applicable to it: `epo:hasUnitCodeListAgencyID, epo:hasUnitCodeListAgencyName, epo:hasUnitCodeListID`.
What does that mean? 
Can one amount be "100 USD" where "USD" is defined by OPOCE, and another be the same "100 USD", but "USD" is defined by the Federal Reserve??

2. `Value` is a more complex class that uses `Amount`, but I think in this way it gives too much freedom to the CurrencyID. 
Given that `hasVATPercentage, hasVATIncludedIndicator` apply to all amounts mentioned in a `Value`,
is it legitimate to vary the CurrencyID in those amounts?

**Comment by VladimirAlexiev** [comment]
> @giorgialodi: Value is currently under revision by the working group with a proposal to use MonetaryValue instead of Value and to use just this class without the need of the class Amount. In any case, in version 2.0.1, Amount is a general amount concept with a value and a currency. Value is a Procurement Value, more domain specific with maximum and minimum value and VAT.

**Comment by eprocurementontology** [comment]
See also the minutes: https://github.com/eprocurementontology/eprocurementontology/wiki/Meeting-Minutes
